opslevel/resource_opslevel_property_assignment.go

```diff
@@ -0,0 +1,124 @@
+package opslevel
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/opslevel/opslevel-go/v2023"
+)
+
+func resourcePropertyAssignment() *schema.Resource {
+	return &schema.Resource{
+		Description: "Manages properties assigned to entities (like Services)",
+		Create:      wrap(resourcePropertyAssignmentCreate),
+		Update:      wrap(resourcePropertyAssignmentUpdate),
+		Read:        wrap(resourcePropertyAssignmentRead),
+		Delete:      wrap(resourcePropertyAssignmentDelete),
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
+			"last_updated": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"definition": {
+				Type:        schema.TypeString,
+				Description: "The custom property definition's ID.",
+				Required:    true,
+				ForceNew:    true,
+			},
```

Comment:
What do you think about adding `ID or alias` to the `Description`s for `definition` and `owner`?

Reply:
Very good point, that should be mentioned. Changed the descriptions to show that.